src/board.rs

```diff
@@ -268,6 +268,21 @@ impl Board {
 
 		return squares;
 	}
+
+	fn find(&self, piece: Piece) -> Vec<&Square> {
+		let mut found_pieces = Vec::new();
+		for row in &self.board {
+			for sq in row {
+				if !sq.is_empty() {
+					if sq.piece.unwrap() == piece { //Because of the implementation of PartialEq on Piece, this will return true
```

Comment:
Now I'm just being a bitch, but this doesn't fit my 80-character-width-Vim-terminal ;)

Reply:
Solved in 7ed45e8b6a65e04787a51851208ba3e240a43095

Reply:
No, still not 80. I've supplied a patch which will make me happy :)

```suggestion
diff --git a/src/board.rs b/src/board.rs
index 254f5f6..fb2280c 100644
--- a/src/board.rs
+++ b/src/board.rs
@@ -274,9 +274,9 @@ impl Board {
    for row in &self.board {
      for sq in row {
        if !sq.is_empty() {
-         if sq.piece.unwrap() == piece { //Because of the implementation of PartialEq
-                         // on Piece, this will return true
-                         //no matter if the pieces have moved or not
+         // because of the implementation of PartialEq on Piece,
+         // this will return true regardless if the pieces have moved or not
+         if sq.piece.unwrap() == piece {
            found_pieces.push(sq);
          }
        }
```

Reply:
cccaa88985cdd4b6a77fda2df0fcd587f4be672c
Happy now? 😳